kirby/api/__init__.py

```diff
@@ -0,0 +1,46 @@
+import requests
+
+from .context import ContextManager, ctx
+
+
+class Kirby:
+    def __init__(self, env_signature, session=None):
+        ctx_manager = ContextManager(env_signature)
+        ctx_manager.load()
+        self.ctx = ctx
+        self._session = session or requests.session()
+
+        self._register()
+
+    def get_topic_id(self, topic_name):
+        result = self._session.get(
+            "/".join([self.ctx.KIRBY_WEB_SERVER, "topic"]),
+            params={"name": topic_name},
+        )
+        assert result.status_code == 200
```

Comment:
Don't do `assert` in code, since you can have them disabled at runtime.

Do 
```python
if result.status_code != 200:
    raise WhateverException('status code blablabla')
```

Reply:
This is now resolved. But I'm not sure about the `There is an issue with the web server`